e2e/app/page/workspace-build-page.ts

```diff
@@ -221,9 +221,9 @@ export const FIELD = {
 };
 
 
-export default class WorkspaceEditPage extends AuthenticatedPage {
+export default abstract class WorkspaceBuildPage extends AuthenticatedPage {
```

Comment:
Renamed to `workspace-build-page.ts` to be consistent with other `*-build-page.ts` files. Changed to `abstract` to avoid accidental usage `new WorkspaceBuildPage()`.